### PR TITLE
fix: Make API messages names be the same as their event

### LINKED ISSUE #79: EventType names do not match actual events messages names
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package version

v1.10.0

### OS

Other

### Language version and compiler version

go1.20.4

### Bug description

Some event messages names do not match their EventType name
For example the message corresponding to `EventType.AccountContactRequestIncomingReceived` is actually called `AccountContactRequestReceived`

This is annoying for writing generic events decoders (it's useful for dumping payloads for example)

I wrote a little script to find inconsistent event naming

```
❯ npx ts-node packages/scripts/checkWeshnetConsistency.ts
Missing event definition for GroupMemberDeviceAdded
Missing event definition for GroupDeviceChainKeyAdded
Missing event definition for AccountContactRequestOutgoingEnqueued
Missing event definition for AccountContactRequestOutgoingSent
Missing event definition for AccountContactRequestIncomingReceived
Missing event definition for AccountContactRequestIncomingDiscarded
Missing event definition for AccountContactRequestIncomingAccepted
Missing event definition for ContactAliasKeyAdded
Missing event definition for MultiMemberGroupAliasResolverAdded
Missing event definition for MultiMemberGroupInitialMemberAnnounced
Missing event definition for MultiMemberGroupAdminRoleGranted
Missing event definition for GroupMetadataPayloadSent
```

### Current behavior

Some event messages names do not match their EventType name

### Expected behavior

All events messages names do match their EventType name

### Environment

macOS 13.3.1

### Other

Since this is an api breaking change I would recommend fixing this before the api is widely consumed

### PR DESCRIPTION
**Pull request by jefft0**
This resolves #79. See the issue for rationale. Change all message names to be the same as the event that sends it. Update the name of related functions that process the event message.

This is a BREAKING CHANGE to the API. Other packages like Berty will need to update the message names.

### CODE REVIEW

**Comment by n0izn0iz** [comment]
LGTM :)

**Review by n0izn0iz — APPROVED** [review]

File: pkg/testutil/filters.go [hunk 1]
```diff
@@ -6,10 +6,10 @@ import (
 	"berty.tech/weshnet/pkg/protocoltypes"
 )
 
-func TestFilterAppMetadata(t *testing.T, events <-chan *protocoltypes.GroupMetadataEvent) []*protocoltypes.AppMetadata {
+func TestFilterAppMetadata(t *testing.T, events <-chan *protocoltypes.GroupMetadataEvent) []*protocoltypes.GroupMetadataPayloadSent {
```
Comment (D4ryl00) [inline pkg/testutil/filters.go:9]: TestFilterAppMetadata -> TestFilterGroupMetadataPayloadSent ?

**Review by D4ryl00 — APPROVED** [review]
LGTM

File: pkg/testutil/filters.go (hunk 1 above)
Comment (jefft0) [reply pkg/testutil/filters.go]: Nice catch. Thanks. I renamed it.

**Pull request merged**